installer.sh

```diff
@@ -1,10 +1,57 @@
 #!/usr/bin/env bash
+
+systemctl status ModemManager | grep 'active (running)' > /dev/null 2>&1
+
+if [ $? != 0 ]
+then
+        sudo systemctl disable ModemManager > /dev/null
+        sudo systemctl stop ModemManager > /dev/null
+fi
+
 set -e
 
 function info { echo -e "[Info] $*"; }
 function error { echo -e "[Error] $*"; exit 1; }
 function warn  { echo -e "[Warning] $*"; }
 
+# Install required dependencies
+test $? -eq 0 || exit 1 "you should have sudo priveledge to run this script"
+
+info ""
+info ""
+info "Installing Home Assistant dependencies"
+info ""
+info ""
+
+while read -r p ; do sudo apt-get install -y $p ; done < <(cat << "EOF"
+	software-properties-common
+	apparmor-utils 
+	apt-transport-https 
+	avahi-daemon 
+	ca-certificates 
+	curl 
+	dbus 
+	jq 
+	network-manager
+EOF
+)
```

Comment:
This runs apt-get for each individual dependency, which should not be needed. Let's make it a single call.